src/services/Results.ts

```diff
@@ -78,6 +80,16 @@ export class Results {
     this.swapDirection()
   }
 
+  updateLogsCount(value: number, relation: string) {
+    if (relation === "eq") {
+      this.logsCount.innerHTML = "Total results = " + value
+    } else if (relation === "gte") {
+      this.logsCount.innerHTML = "Found at least " + value + " results"
+    } else {
```

Comment:
nit: I would go with more symmetry here. Something like this:

```suggestion
    if (relation === "eq") {
      this.logsCount.innerHTML = "Found exactly " + value + "results"
    } else if (relation === "gte") {
      this.logsCount.innerHTML = "Found at least " + value + " results"
    } else {
```